src/SFX.cpp

```diff
@@ -0,0 +1,45 @@
+#include "SFX.h"
+
+#include <iostream>
+#include <algorithm>
+#include <cmath>
+
+namespace ccm
+{
+	SFX::SFX(std::filesystem::path filepath)
+	{
+		if (!std::filesystem::exists(filepath))
+		{
+			std::cerr << "Could not find: " << filepath << '\n';
+		}
```

Comment:
We might consider returning here since it's pretty much guaranteed to not load the `m_chunk` anyway. Remember to default it to `nullptr`

Reply:
Yeah, I'll return early. Doing `Mix_Music* m_music{}` (uniform initialization) should already set it to `nullptr`.

Reply:
Right. My bad!